src/black/__init__.py

```diff
@@ -644,12 +652,17 @@ def reformat_one(
 
 
 def reformat_many(
-    sources: Set[Path], fast: bool, write_back: WriteBack, mode: Mode, report: "Report"
+    sources: Set[Path],
+    fast: bool,
+    write_back: WriteBack,
+    mode: Mode,
+    report: "Report",
+    workers: Optional[int],
```

Comment:
```suggestion
    worker_count: int = os.cpu_count(),
```

This way we could just explicitly send in the worker_count as an argument.

Only worry is potentially people using the API could break - so a default argument helps there ...